index.js

```diff
@@ -22,7 +27,9 @@ module.exports = {
 
   init: function() {
     var bowerDeps = this.project.bowerDependencies();
+
     var VersionChecker = require('ember-cli-version-checker');
+    var options = this.options = this.options || {};
```

Comment:
ya that works in some cases, but actually it should be done at 105, submiting PR